src/sampling.jl

```diff
@@ -0,0 +1,219 @@
+using StatsBase
+
+function _sample(
```

Comment:
We might have to check that sampling is well-defined by checking site type or smth. Or at least say that we assume quantum-state like sampling, i.e., according to the absolute square of the encoded coefficients. This will not properly sample from Pauli tensor networks.